### ISSUE TITLE
[Debug mode] 7 duplicate symbols for architecture x86_64

### ISSUE THREAD

**Issue by ttDemon** [duyluonglc/react-native-gvr#2]
I have a problem when I run release is ok but if I run debug it has the error as below. I do not understand why. Please tell me if possible.

And another issue is when I run release, I watch video, when load video it shows a pink screen, how can I configure?
![screen shot 2018-05-10 at 10 37 05 am](https://user-images.githubusercontent.com/26439136/39851353-9285bd06-5440-11e8-91cb-6aa75a2685e2.png)

**Comment by duyluonglc** [comment]
did you forget this code in podfile ?
```
# Execute every pod install
  post_install do |installer|
      system(". ./pod_post_install.sh")
  end
```

**Comment by ttDemon** [comment]
Yes, I have done it as a guide, I still do not understand why build release is still debug build is error
